docs/about/history.rst

```diff
@@ -23,12 +23,22 @@ Breaking
 
       -  Set minimum render pipeline package version to 10.5. `[HDRP] [URP]`
 
+   .. only:: hdrp
+
+      -  *Underwater Post-Processing* is disabled by default which means it will be inactive if the *Underwater Volume Override* is not present in the scene. `[HDRP]`
+
    .. only:: urp
 
       -  Remove *Sample Shadows* Render Feature as it is now scripted.
          Unity will raise a missing Render Feature reference error.
          Remove the missing Render Feature to resolve. `[URP]`
 
+Preview
```

Comment:
I think since we're deprecating the old stuff this should be in the Changed section as it's not an optional/for testing preview thing

Reply:
Sure. Done.